### PR TITLE
Added ESIP tutorial notebook

### PR DESCRIPTION
**Pull request by lsterzinger**
I've gotten good feedback on the notebook I presented at ESIP's winter meeting, thought it might be worth including in the docs. I added it to the bottom of the "quick start" page but if you think there's a better place (or if we should have a dedicated page for interactive guides or something) I'm happy to move it

### CODE REVIEW

File: docs/source/test_example.rst [hunk 1]
```diff
@@ -93,3 +93,8 @@ examples).
 Note that, if the combining was done previously and saved to a JSON file, then the path to
 it should replace ``out``, above, along with a ``target_options`` for any additional
 arguments fsspec might to access it
+
+Example/Tutorial Notebook
+================
```
Comment (martindurant) [inline docs/source/test_example.rst:98]:
```suggestion
Example/Tutorial Notebook
====================
```

**Comment by martindurant** [comment]
Glad to include this link. Were there any output datasets that you would like to add to the "case studies"?

**Comment by lsterzinger** [comment]
I am not aware of any other than what's already there, though a colleague of mine has been kerchunking a year or more of GOES-16 cloud mask data for a research project. If we can find some hosting I'm sure she'd be happy to share the files

**Comment by martindurant** [comment]
>  If we can find some hosting

You mean the output JSON? Yes, of course we can put them somewhere. Please cc the colleague of yours here, or get in touch on gitter/slack/email.

**Pull request merged**